packages/design-system/src/components/Accordion/index.tsx

```diff
@@ -33,18 +33,21 @@ const AccordionComponent: React.FC<AccordionProps> = ({
                   iconPosition={iconPosition}
                   icon={
                     <ChevronDownSvg
-                      className={classNames("w-accordion-widthChevron transform transition", {
-                        "rotate-0": isExpanded,
-                        "-rotate-90": !isExpanded,
-                      })}
+                      className={classNames(
+                        "w-accordion-chevron transform transition",
+                        {
+                          "rotate-0": isExpanded,
+                          "-rotate-90": !isExpanded,
+                        }
+                      )}
                     />
                   }
                 />
               ) : (
                 <Accordion.Icon
                   iconPosition={iconPosition}
                   icon={
-                    <div className="relative flex w-accordion-widthMinus items-center justify-center">
+                    <div className="w-accordion-minus relative flex items-center justify-center">
```

Comment:
Shouldn't it be here? 
```suggestion
                    <div className="w-accordion-wMinus relative flex items-center justify-center">
```

Reply:
no, beacause we already know from "w" at the beggining of the class, that it's width

Reply:
with border radius it's necessary beacause its border-accordion-sth both for width and radius